tensorflow_addons/text/skip_gram_ops.py

```diff
@@ -19,27 +19,30 @@
 
 from tensorflow_addons.utils.resource_loader import LazySO
 
+from tensorflow_addons.utils.types import AcceptableDTypes, FloatTensorLike, TensorLike
+from typing import Optional
+
 _skip_gram_so = LazySO("custom_ops/text/_skip_gram_ops.so")
 
 tf.no_gradient("Addons>SkipGramGenerateCandidates")
 
 
 def skip_gram_sample(
-    input_tensor,
-    min_skips=1,
-    max_skips=5,
-    start=0,
-    limit=-1,
-    emit_self_as_target=False,
-    vocab_freq_table=None,
-    vocab_min_count=None,
-    vocab_subsampling=None,
-    corpus_size=None,
-    batch_size=None,
-    batch_capacity=None,
-    seed=None,
-    name=None,
-):
+    input_tensor: TensorLike,
+    min_skips: FloatTensorLike = 1,
+    max_skips: FloatTensorLike = 5,
+    start: FloatTensorLike = 0,
+    limit: FloatTensorLike = -1,
+    emit_self_as_target: bool = False,
+    vocab_freq_table: AcceptableDTypes = None,
```

Comment:
It seems to be a lookup table, not a data type.

Reply:
Thanks @gabrieldemarmiesse. I added `tf.lookup.KeyValueTensorInitializer` in `AcceptableDTypes`.  Maybe just place here directly.

Reply:
Indeed. Having `vocab_freq_table=tf.float32` wouldn't be correct for exemple.

Reply:
Got it. Thanks @gabrieldemarmiesse.